### PR TITLE
Eq 1450 section introduction pages

### PR DESCRIPTION
**Pull request by shahi645**
### What is the context of this PR?
Added a section introduction template, and implemented into `census_household`. (Who lives here, Household and accommodation, Household member, Visitors)

### How to review 
Check trello card:
https://trello.com/c/XhmyZOo1/1450-census-section-content
Does changes satisfy this card

### CODE REVIEW

**Review by charlotte-hirst — COMMENTED** [review]
Happy from a content point of view

**Comment by shahi645** [comment]
@charlotte-hirst Good news, content wont change, just changing the logic behind it

File: tests/integration/census/test_census_household_submission_data.py [hunk 1]
```diff
@@ -1118,11 +1101,12 @@ def complete_individual_section_person_2(self):
         # Completed Individual section
         self.assertInPage('There are no more questions for Anjali Yo')
         self.post(action='save_continue')
-        self.assertInPage('The next section covers questions for Visitor 1')
+        self.assertInPage('Visitors')
```
Comment (jonnyshaw89) [inline tests/integration/census/test_census_household_submission_data.py]: This is probably a bad assert as this value on in the navigation?

**Review by jonnyshaw89 — CHANGES_REQUESTED** [review]

File: tests/integration/census/test_census_household_submission_data.py (hunk 1 above)
Comment (shahi645) [reply tests/integration/census/test_census_household_submission_data.py]: Agreed will change

**Review by yoganandkunche — APPROVED** [review]
LGTM :+1:

File: tests/integration/census/test_census_household_submission_data.py [hunk 2]
```diff
@@ -1118,11 +1101,12 @@ def complete_individual_section_person_2(self):
         # Completed Individual section
         self.assertInPage('There are no more questions for Anjali Yo')
         self.post(action='save_continue')
-        self.assertInPage('The next section covers questions for Visitor 1')
+        self.assertInPage('Information you need')
```
Comment (jonnyshaw89) [inline tests/integration/census/test_census_household_submission_data.py]: This value still isn't specific to just this page.

**Review by jonnyshaw89 — CHANGES_REQUESTED** [review]

**Review by jonnyshaw89 — APPROVED** [review]

**Comment by jonnyshaw89** [comment]
@shahi645 Squishy Squashy?

**Review by ajmaddaford — CHANGES_REQUESTED** [review]
@charlotte-hirst I don't think the section title should be "Information about {name}" as:
- the page doesn't contain information about {name}
- the title doesn't match the navigation

I think the title should just be "{name}".

**Comment by charlotte-hirst** [comment]
Agree with andrew to change to use the name as the section title @shahi645

**Review by ajmaddaford — APPROVED** [review]

**Pull request merged**